modules/web/src/app/settings/admin/seed-configurations/seed-configurations-details/component.ts

```diff
@@ -80,6 +80,32 @@ export class SeedConfigurationDetailsComponent implements OnInit {
       });
   }
 
+  onSortChange(sort: Sort): void {
```

Comment:
Can we use `sortingDataAccessor` instead of manually sorting?
For example,

```
this.dataSource.sortingDataAccessor = (item, property) => {
      switch (property) {
        case Column.Provider:
          return item.provider;
        case Column.Datacenters:
          return item.datacentersCount;
        case Column.Clusters:
          return item.clustersCount;
        default:
          return item[property];
      }
    };

```

For reference take a look at `src/app/dynamic/enterprise/quotas/component.ts` `_setSortConfig` method.